src/components/ui/EditIcon/index.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import { BLACK } from '../../../settings/__color';
+import { fontSizes } from '../../../settings/__fonts';
+
+const EditIcon = () => {
+  return (
+    <EditIcon.Wrapper>
+      <i className="icon ion-md-create" />
+    </EditIcon.Wrapper>
+  );
+};
```

Comment:
Nice, can we just make this an Icon which accepts an Icon type and displays that Icon?
Also, we should have a story for this.
This should be a regular function, not an arrow function for performance.